frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,74 @@
+import { useRef } from "react";
+import './Signup.css';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const Login = ({setCurrUser, setShow}) =>{
+  const formRef=useRef();
+  const login=async (userInfo, setCurrUser)=>{
+    const url="http://localhost:3000/login";
```

Comment:
Declare this constant outside this component with a constant variable naming convention i.e 
``` suggestion
    const BASE_URL = 'http://localhost:3000';
    const LOGIN_URL = `/${BASE_URL}/login`;
```
and use this `LOGIN_URL`.

Reply:
Okay Done Sir.